Features/Context/SubContext/CommonActions.php

```diff
@@ -12,6 +12,91 @@
 trait CommonActions
```

Comment:
I'd really prefer if we could aim at keeping those methods in dedicated contexts, in order to sandbox things. But not now.